### PR TITLE
fix: remove tooltip, fix sizing and alignment on deskotp

### PR DESCRIPTION
**Pull request by hyperobject**
Removes unneeded tooltips for custom emojis on desktop and makes sizing a little more consistent with stock emoji.

### CODE REVIEW

**Comment by mmaxim** [comment]
This likely conflicts with @songgao's alias PR, we should probably try to resolve some way.

**Comment by songgao** [comment]
I was hoping we could maintain a fixed ratio between stock emojis and custom emojis. For example, if we use a ratio of 0.85, when stock emoji renders at 32 custom emojis will render at 27.2. Here's my reasoning: the reason why we need to treat them differently is stock emoji doesn't render to the same size as their font size in pixels, but custom emojis do. But since they scale up and down proportionally, there's no reason the ration between stock and custom emojis can change. With that thought, I have used a fixed factor of 0.85 for deriving custom emoji size from stock emoji size.

Though I see in this PR that the custom emojis sometimes get larger size, so it seems my assumptions might have been wrong. My PR unfortunately changes how we choose component to render emojis and specify sizes (via an exported function `renderEmoji` in `util/emoji.tsx`) which will conflict with this. But please feel free to merge this and we can work from there and incorporate the changes into my PR. Let's also talk about how we should plumb the size parameter (and when they are needed).

File: shared/chat/conversation/messages/react-button/emoji-picker/index.tsx [hunk 1]
```diff
@@ -226,7 +226,7 @@ class EmojiPicker extends React.PureComponent<Props, State> {
         key={emoji.short_name}
       >
         {emoji.source ? (
-          <Kb.CustomEmoji size="Medium" src={emoji.source} alias={emoji.short_name} />
+          <Kb.CustomEmoji size="MediumLarge" src={emoji.source} />
```
Comment (songgao) [inline shared/chat/conversation/messages/react-button/emoji-picker/index.tsx:229]: Does mobile need to be different? `singleEmojiWidht` (used below for stock emojis) is different on mobile vs desktop.

Comment (hyperobject) [reply shared/chat/conversation/messages/react-button/emoji-picker/index.tsx:229]: This PR doesn't handle mobile changes, was still running into some alignment issues on ios and android so I figured i'd just get desktop changes in for now

**Pull request merged**